tickets.md

```diff
@@ -2,4 +2,4 @@
 title: Tickets
 ---
 
-Tickets are not on sale yet.
+Tickets are now on sale at: https://www.eventbrite.com/e/lambdale-2019-tickets-63598072547
```

Comment:
Does Jekyll turn that into a link automatically or should we make it a markdown link?